### PR TITLE
Viro AUStation Sensibilities Reconfigure 

### PR DESCRIPTION
**Pull request by MrLostman**
## About The Pull Request
**Bee Infestation**
-symbiotic and honey producing have been separated
`resistance 6 and 8 respectively`
-bee spawn chance linked to stage speed
`prob(25 + clamp(A.properties["stage_rate"] * 5, -25, 70)`
-can stop it from making bees but removes royal jelly,and halfs the amount of honey
`resistance 12`
**Pituitary Disruption**
-no longer spawns meta grudges
-no longer makes you vomit all over the station
-no longer randomises organs
**Nanite Symbiosis**
-no longer causes glitches
**Self Resp**
-farting removal modularized
**Viral Aggressive Meta**
-If it cures the virus does not make them immune
**Weight Loss**
-threshold to make it stop you from becoming fat
`resistance 8`
**Spiked Skin**
-returned to 60 armour cap
**Deoxyribonucleic Acid Saboteur**
-removed bad mutations from good mutation threshold
-removed stealth threshold

### Stat Changes
stealth, resistance, stage speed, trans
**Deafness**
-1,-2,-1,-3 -> 0, -2, -1, -1
**Fever**
-1,3,3,2 -> 0,3,3,2
**Spontaneous Combustion**
1,-2,-3,-3 -> 1,-1,-2,-1
**Necrotizing Fasciitis**
-3,-3,0,-4 -> -1,-1,0,-1
**Narcolepsy**
1,-2,-3,-2 -> 1,-1,-2,-1
**Hyphema**
-1,-3,-4,-2 -> -1,-2,1,-1
**Vomiting**
-2,1,0,2 -> -3,2,1,3
**Deoxyribonucleic Acid Saboteur**
-2,-3,0,-3 -> -1,-2,0,-2

## Changelog
:cl:
balance: rebalanced stats of deafness, fever, spontaneous combustion, necrotizing fasciitis, narcolepsy, hyphema and vomiting, DAS
refactor: modularized self resp fart removal
tweak: bee infestation symbiotic and honey producing have been separated, bee spawn chance linked to stage speed and a no bee threshold
tweak: pituitary disruption, superficial healing and nanite symbiosi has had zesko grief code removed
tweak: viral aggressive meta doesn't make host immune if it cures
tweak: weight loss threshold that stops people from getting fat
tweak: spiked skin armour reverted to 60
tweak: das stealth threshold removed and bad mutations removed from good threshold
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
> tweak: spiked skin armour reverted to 60


lol

File: austation/code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		M.adjustFireLoss(-power)
+		healed = TRUE
+
+	if(M.getToxLoss() && M.getToxLoss() <= threshhold)
+		M.adjustToxLoss(-power)
+	return healed
+
+/datum/symptom/growth
+	threshold_desc = "<b>Stage Speed 6:</b> The disease heals brute damage at a fast rate.<br>\
+					<b>Stage Speed 12:</b> The disease heals brute damage incredibly fast, but deteriorates cell health and causes tumors to become more advanced. The disease will also regenerate lost limbs"
```
Comment (Dingo-Dongler) [inline austation/code/datums/diseases/advance/symptoms/heal.dm:18]:
```suggestion
					<b>Stage Speed 12:</b> The disease heals brute damage incredibly fast, but deteriorates cell health. The disease will also regenerate lost limbs"

```

**Review by MCterra10 — CHANGES_REQUESTED** [review]
`code/datums/diseases/advance/symptoms/oxygen.dm`
please ensure you are not changing line endings

**Comment by MrLostman** [comment]
merge it

**Comment by TheFakeElon** [comment]
read Terra's review smh, you haven't fixed your line endings

**Pull request closed without merging**